src/main/java/org/team1540/robot2025/RobotContainer.java

```diff
@@ -129,65 +129,80 @@ private void configureButtonBindings() {
-                .and(buttonBoard.flexFalse())
-                .and(driver.leftStick())
-                .whileTrue(AutoAlignCommands.alignToBranch(FieldConstants.ReefBranch.B, drivetrain));
-        buttonBoard
-                .branchFaceAt(ReefButton.C)
-                .and(buttonBoard.flexFalse())
-                .and(driver.leftStick())
-                .whileTrue(AutoAlignCommands.alignToBranch(FieldConstants.ReefBranch.C, drivetrain));
-        buttonBoard
-                .branchFaceAt(ReefButton.D)
-                .and(buttonBoard.flexFalse())
-                .and(driver.leftStick())
-                .whileTrue(AutoAlignCommands.alignToBranch(FieldConstants.ReefBranch.D, drivetrain));
-        buttonBoard
-                .branchFaceAt(ReefButton.E)
-                .and(buttonBoard.flexFalse())
-                .and(driver.leftStick())
-                .whileTrue(AutoAlignCommands.alignToBranch(FieldConstants.ReefBranch.E, drivetrain));
-        buttonBoard
-                .branchFaceAt(ReefButton.F)
-                .and(buttonBoard.flexFalse())
-                .and(driver.leftStick())
-                .whileTrue(AutoAlignCommands.alignToBranch(FieldConstants.ReefBranch.F, drivetrain));
-        buttonBoard
-                .branchFaceAt(ReefButton.G)
-                .and(driver.leftStick())
-                .whileTrue(AutoAlignCommands.alignToBranch(FieldConstants.ReefBranch.G, drivetrain));
-        buttonBoard
-                .branchFaceAt(ReefButton.H)
-                .and(buttonBoard.flexFalse())
-                .and(driver.leftStick())
-                .whileTrue(AutoAlignCommands.alignToBranch(FieldConstants.ReefBranch.H, drivetrain));
-        buttonBoard
-                .branchFaceAt(ReefButton.I)
-                .and(buttonBoard.flexFalse())
-                .and(driver.leftStick())
-                .whileTrue(AutoAlignCommands.alignToBranch(FieldConstants.ReefBranch.I, drivetrain));
-        buttonBoard
-                .branchFaceAt(ReefButton.J)
-                .and(buttonBoard.flexFalse())
-                .and(driver.leftStick())
-                .whileTrue(AutoAlignCommands.alignToBranch(FieldConstants.ReefBranch.J, drivetrain));
-        buttonBoard
-                .branchFaceAt(ReefButton.K)
-                .and(buttonBoard.flexFalse())
-                .and(driver.leftStick())
-                .whileTrue(AutoAlignCommands.alignToBranch(FieldConstants.ReefBranch.K, drivetrain));
-        buttonBoard
-                .branchFaceAt(ReefButton.L)
-                .and(buttonBoard.flexFalse())
-                .and(driver.leftStick())
-                .whileTrue(AutoAlignCommands.alignToBranch(FieldConstants.ReefBranch.L, drivetrain));
+        for (ButtonBoard.ReefButton button : ButtonBoard.ReefButton.values()) {
+            for (FieldConstants.ReefHeight height : FieldConstants.ReefHeight.values()) {
+                buttonBoard
+                        .branchFaceAt(button)
```

Comment:
Doesn't need to happen now but we might want to cache the buttonboard triggers like the official wrappers do